src/actionlist/action-list-item.scss

```diff
@@ -34,30 +34,50 @@
   }
 
   // only hover li without list as children
-  &:not(.ActionList-item--hasSubItem) {
+  &:not(.ActionList-item--hasSubItem),
+  // target contents of first child li if sub-item (li wraps item label + nested ul)
+  &.ActionList-item--hasSubItem > .ActionList-content {
     &:hover {
       cursor: pointer;
       background-color: var(--color-action-list-item-default-hover-bg);
+
+      // stylelint-disable-next-line selector-max-specificity
+      &:not(.ActionList-item--navActive) {
+        // Support for "Windows high contrast mode"
+        outline: $border-style $border-width transparent;
+        outline-offset: -$border-width;
+        // stylelint-disable-next-line primer/box-shadow
+        box-shadow: inset 0 0 0 2px var(--color-action-list-item-default-active-border);
+      }
     }
 
     &:active {
       background: var(--color-action-list-item-default-active-bg);
 
+      // stylelint-disable-next-line selector-max-specificity
+      &:not(.ActionList-item--navActive) {
+        // Support for "Windows high contrast mode" https://sarahmhigley.com/writing/whcm-quick-tips/
+        outline: $border-style $border-width transparent;
+        outline-offset: -$border-width;
+        // stylelint-disable-next-line primer/box-shadow
+        box-shadow: inset 0 0 0 2px var(--color-action-list-item-default-active-border);
+      }
+
       @media screen and (prefers-reduced-motion: no-preference) {
         animation: ActionList-item-active-bg 4s forwards cubic-bezier(0.33, 1, 0.68, 1);
       }
 
       // stylelint-disable primer/box-shadow
       @keyframes ActionList-item-active-bg {
-        // stylelint-disable-next-line max-nesting-depth
+        // stylelint-disable-next-line max-nesting-depth, selector-max-specificity
         50% {
-          box-shadow: inset 0 0 0 rgba(#000, 0.04);
+          // this could work if primitive supported rgb (not valid right now)
```

Comment:
I think we have to compromise on the fun interaction here for now. We could stack box-shadows to ensure we have a contrast border around the shadow that makes the item feel like its shrinking, but its not valid to use `rgba()` on a CSS variable that resolves as a hex. It would need to be `rgb`. 😞

Reply:
@langermank sorry I missed this comment!

I think we could remove the inset `box-shadow` while keeping the scaling interaction in that case :)

Reply:
@vdepizzol no worries! Okie, I think I got it 👍